### PR TITLE
Engdocs 1522

### PR DESCRIPTION
**Pull request by aevesdocker**
This PR tidies networks.md also adds a meatier example taken from elsewhere in docs

also closes https://github.com/compose-spec/compose-spec/pull/368 and https://github.com/compose-spec/compose-spec/pull/390

### CODE REVIEW

File: 06-networks.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 ## Networks top-level element
 
-Networks are the layer that allow services to communicate with each other. The networking model exposed to a service
-is limited to a simple IP connection with target services and external resources, while the Network definition allows
-fine-tuning the actual implementation provided by the platform.
+Networks are the layer that allow services to communicate with each other.
 
-Networks can be created by specifying the network name under a top-level `networks` section.
-Services can connect to networks by specifying the network name under the service [`networks`](05-services.md#networks) subsection
+The top-level `networks` element lets you configure named networks that can be reused across multiple services. To use a network across multiple services, you must explicitly grant each service access by using the [networks](05-services.md) attribute. The `netowrks` top-level element has additional syntax that provides more granular control.
```
Comment (glours) [inline 06-networks.md:5]:
```suggestion
The top-level `networks` element lets you configure named networks that can be reused across multiple services. To use a network across multiple services, you must explicitly grant each service access by using the [networks](05-services.md) attribute. The `networks` top-level element has additional syntax that provides more granular control.
```

**Review by glours — APPROVED** [review]
Just one minor typo to fix otherwise LGTM 👍

**Comment by otkd** [comment]
@aevesdocker Noticed most of my changes from https://github.com/compose-spec/compose-spec/pull/390/commits/ee89e79bcfcc299053dd8a4db64a9ebfd50f6fbe were included in https://github.com/compose-spec/compose-spec/pull/388/commits/57fc6e1dbf21d2cba604ef1386dd8b2b00d78376, any reason to exclude example for internal network?

**Review by laurazard — APPROVED** [review]
LGTM

**Pull request merged**